charts/victoria-metrics-cluster/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: 1.29.3
 description: Victoria Metrics Cluster version - high-performance, cost-effective and scalable TSDB, long-term remote storage for Prometheus
 name: victoria-metrics-cluster
-version: 0.1.0
+version: 0.1.1
```

Comment:
the latest version is 0.2.0 could please bump to 0.3.0

Reply:
Done